ils_middleware/tasks/folio/build.py

```diff
@@ -103,8 +103,8 @@ def _instance_type_id(**kwargs) -> tuple:
     folio_client = kwargs["folio_client"]
     values = kwargs["values"]
 
-    # Only use first value
-    name = values[0][0]
+    # Only use first value and lowercase
+    name = values[0][0].lower()
```

Comment:
so, i was trying to trace through the invocation of this via its application in `_inventory_record` via `transforms.get`, but i'm a bit unfamiliar with this code and couldn't figure out what i was looking for...  will the `row["name"]` value used for comparison below already always be lower case?  is the point here to effectively do a case insensitive comparison?